scripts/corelli/calibration/utils.py

```diff
@@ -7,12 +7,14 @@
 
 from os import path
 import numpy as np
+from numpy.polynomial.polynomial import polyval
 from typing import List, Optional, Union
```

Comment:
Not sure if `List` is used as a type

Reply:
it's not a built-in type, if that's what you mean, but `List` it's a declaration of built-in type `list`

Reply:
yup, I mean type a variable type `list::List[int]`. I'm just used to seeing types everywhere, not a big deal for this.